### PR TITLE
Feature/mockup backend interfaces

### PR DESCRIPTION
**Pull request by tuansondinh**
interfaces for hmi-evaluation project
- for communication/handling data between mockup-backend and frontend

### CODE REVIEW

File: src/mockup-backend-evaluation/interfaces.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    timestamp: Date;
+    message: string;
+    id: string;
+    status: string;
+    name: string;
+}
+
+export interface PlantInfoInterface {
+    scenarioRunning: boolean;
+    receiptRunning: boolean;
```
Comment (julorenz) [inline src/mockup-backend-evaluation/interfaces.ts]: recipe

File: src/mockup-backend-evaluation/interfaces.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    plantCleaned: boolean;
+    currentScenario: string;
+}
+
+export enum Status {
```
Comment (julorenz) [inline src/mockup-backend-evaluation/interfaces.ts:39]: None = 'No Status available',
Normal = 'Normal',
PreWarning = 'PreWarning',
Warning = 'Warning',
Alert = 'Alert',
Malfunction = 'Malfunction'

**Review by julorenz — CHANGES_REQUESTED** [review]

File: src/mockup-backend-evaluation/interfaces.ts (hunk 1 above)
Comment (tuansondinh) [reply src/mockup-backend-evaluation/interfaces.ts]: done

File: src/mockup-backend-evaluation/interfaces.ts (hunk 2 above)
Comment (tuansondinh) [reply src/mockup-backend-evaluation/interfaces.ts:39]: done, raised to alpha 15. This change will affect mockup-backend and frontend, which is not handled yet, so they still use alpha14.

**Pull request merged**